tests/e2e/utils/wait_for_ols.py

```diff
@@ -0,0 +1,54 @@
+"""This module contains the wait_for_ols function and generate_junit_report function.
+
+to check the readiness of the OLS and generate a JUnit report.
+"""
+
+import sys
+import time
+import warnings
+from pathlib import Path
+
+import requests
+from requests.exceptions import SSLError
+from urllib3.exceptions import InsecureRequestWarning
+
+sys.path.append(Path(__file__).resolve().parents[1].as_posix())
```

Comment:
another thing i would not expect to be needed, especially now that this is a utility/helper, not a standalone script invocation.

Reply:
hmm +1 tbh

Reply:
I still think this can be removed and should be